sceptre/sageit/templates/accounts.yaml

```diff
@@ -56,7 +56,7 @@ Resources:
           -
             Effect: Allow
             Principal:
-              Service: ec2.amazonaws.com
+              - Service: ec2.amazonaws.com
```

Comment:
this makes `Service` a list which i think is wrong.  examples here: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-iam-role.html#aws-resource-iam-role--examples

Reply:
Fixed, thought that principal could be a list of different types...